packages/asc-ui/src/components/Footer/FooterBottom.tsx

```diff
@@ -3,6 +3,10 @@ import styled from 'styled-components'
 import { themeSpacing } from '../../utils'
 import { Row } from '../Grid'
 
+type Props = {
+  rowProps: object
```

Comment:
There should be a better type that we can use here. Does the row itself not have some props?

Reply:
Row has plenty of props. I just wanted to be able to pass my own `style` prop to pass my own styles. I don't really need those other props, although they can be helpful for someone else.
```
children: React.ReactNode
  className?: string
  debug?: boolean
  debugColor?: string
  halign?: FlexJustify
  hasMargin?: boolean
  hasMaxWidth?: boolean
  valign?: TypeFlexPosition
```

NB: I'm fine with closing this PR because I've found another solution in my own project, even though I still find it strange that FooterBottom has a Row as parent.

ping: @chrisvanmook